pkg/kubectl/scale.go

```diff
@@ -167,6 +153,7 @@ func (precondition *ScalePrecondition) ValidateReplicationController(controller
 	return nil
 }
 
+// TODO(p0lyn0mial): remove ReplicationControllerScaler
```

Comment:
Being held for the reapers?  If so, open an issue.

Reply:
Yes, that is correct. I was planning to deal with the reapers in the very next pull. In that case do we need an issue for that ?

Unless your plan is to leave the reapers for the time being and focus on preparing kubectl to support scaling custom resources.